backend/executor.go

```diff
@@ -404,8 +404,8 @@ loop:
 }
 
 // mustEmbedUnimplementedTaskHubSidecarServiceServer implements protos.TaskHubSidecarServiceServer
-func (grpcExecutor) mustEmbedUnimplementedTaskHubSidecarServiceServer() {
-}
+func (grpcExecutor) mustEmbedUnimplementedTaskHubSidecarServiceServer() { //nolint:unused
```

Comment:
No, this is not how this method is meant to be used.

This is here because gRPC wants you to embed the unimplemented `TaskHubSidecarServiceServer` struct...

Reply:
I didn't change anything here... I only added the linter comment